src/current/cockroachcloud/provision-a-cluster-with-terraform.md

```diff
@@ -167,220 +70,109 @@ In this tutorial, you will create a CockroachDB {{ site.data.products.dedicated
 
     Enter `yes` when prompted to apply the plan and create the cluster.
 
-You will see output similar to the following:
+Terraform reports the actions it will take. Verify the details, then type `yes` to apply the changes.
+{% endcapture %}
+
+Terraform uses a infrastructure-as-code approach to managing resources. Terraform configuration files allow you to define resources declaratively and let Terraform manage their lifecycle.
```

Comment:
uses "an" infrastructure-as-code, rather than "a"?

Reply:
This content is actually entirely redundant with the first paragraph of the intro. Removed.